src/wallet/wallet.cpp

```diff
@@ -5240,6 +5244,7 @@ CAmount CWallet::GetLegacyBalance(const isminefilter& filter, int minDepth) cons
 }
 
 void CWallet::AvailableCoins(vector<COutput>& vCoins,
```

Comment:
Decide and document what "available" means when `asOfHeight` is set and `nMinDepth == 0`.

Reply:
This now has an `assert` to ensure that `nMinDepth > 0` when `asOfHeight`.